docs/academy/module5.md

```diff
@@ -73,6 +73,11 @@ Learn more about the Firmware Update Object on the [OMA LwM2M Object and Resourc
     | `update result 10`  | **Firmware update cancelled** |
     | `update result 11`  | **Firmware update deferred** |
 
+
+## What is Bootloader and why do I need this here?
```

Comment:
No no no, this place is `z dupy` for that in my opinion. THis hsould be in Exercise 5, before `Implement Firmware Update` for example.